zebra-chain/src/transaction/tests/vectors.rs

```diff
@@ -156,6 +167,25 @@ fn empty_v4_round_trip() {
     assert_eq!(data, data2, "data must be equal if structs are equal");
 }
 
+/// Check if an empty V5 transaction can be deserialized by librustzcash too.
+#[test]
+fn empty_v5_librustzcash_round_trip() {
+    zebra_test::init();
+
+    let tx: &Transaction = &*EMPTY_V5_TX;
+    let data = tx.zcash_serialize_to_vec().expect("tx should serialize");
+
+    let branch_id: u32 = NetworkUpgrade::Nu5
+        .branch_id()
+        .expect("Network upgrade must have a Branch ID")
+        .into();
+    let branch_id: zcash_primitives::consensus::BranchId = branch_id
+        .try_into()
+        .expect("zcash_primitives and Zebra have the same branch ids");
+    let _alt_tx = zcash_primitives::transaction::Transaction::read(&data[..], branch_id)
+        .expect("librustzcash deserialization must work for zebra serialized transactions");
```

Comment:
Let's make it clear that this test might not work:
```suggestion
        .expect("librustzcash deserialization might work for empty zebra serialized transactions. Hint: if empty transactions fail, but other transactions work, delete this test");
```